src/Templates/DotnetCore/Microsoft.NetMaui.CSharp.ProjectTemplates/source.extension.vsixmanifest

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
+  <Metadata>
+    <Identity Id="Microsoft.NetMaui.CSharp.ProjectTemplates" Version="|%CurrentProject%;GetVsixVersion|" Language="en-US" Publisher="Microsoft" />
+    <DisplayName>C# .NET MAUI Project Templates</DisplayName>
+    <Description>C# project templates for .NET MAUI.</Description>
+  </Metadata>
+  <Installation Experimental="true">
+    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[17.0,]">
```

Comment:
I think that [17.0,) is preferred - e.g. parenthesis instead of square bracket to close. However I see that the other .vsixmanifests are doing it this way so I don't have a strong opinion here.

Reply:
Agreed, I usually see the parenthesis too.  But I'll just follow convention in this repo.  I think that since there's no second value both ) and ] are functionally equivalent.